custom_components/thermal_comfort/translations/en.json

```diff
@@ -0,0 +1,48 @@
+{
+  "options": {
+    "step": {
+      "init": {
+        "title": "Thermal comfort settings",
+        "data": {
+          "temperature_sensor": "Temperature sensor",
+          "humidity_sensor": "Humidity sensor",
+          "poll": "Do you want the sensors to be polled? This can avoid double calculated values if your input sensors split change updates for humidity and temperature.",
+          "absolutehumidity": "Enable absolute humidity sensor by default?",
+          "dewpoint": "Enable dew point sensor by default?",
+          "frostpoint": "Enable frost point sensor by default?",
+          "frostrisk": "Enable frost risk sensor by default?",
+          "heatindex": "Enable heat index sensor by default?",
+          "simmerindex": "Enable simmer index sensor by default?",
+          "simmerzone": "Enable simmer zone sensor by default?",
+          "perception": "Enable perception sensor by default?",
+          "show_advanced_options": "Advanced options"
+        }
+      }
+    }
+  },
+  "config": {
+    "abort": {
+      "already_configured": "This combination of temperature and humidity sensors is already configured"
+    },
+    "step": {
+      "user": {
+        "title": "Thermal comfort settings",
+        "data": {
+          "name": "Name",
+          "temperature_sensor": "Temperature sensor",
+          "humidity_sensor": "Humidity sensor",
+          "poll": "Do you want the sensors to be polled? This can avoid double calculated values if your input sensors split change updates for humidity and temperature.",
+          "absolutehumidity": "Enable absolute humidity sensor by default?",
+          "dewpoint": "Enable dew point sensor by default?",
+          "frostpoint": "Enable frost point sensor by default?",
+          "frostrisk": "Enable frost risk sensor by default?",
+          "heatindex": "Enable heat index sensor by default?",
+          "simmerindex": "Enable simmer index sensor by default?",
+          "simmerzone": "Enable simmer zone sensor by default?",
+          "perception": "Enable perception sensor by default?",
+          "show_advanced_options": "Advanced options"
+        }
+      }
+    }
+  }
+}
```

Comment:
I would reduce this to `Enable x` strings. e.g. `Enable Polling`, `Enable Dew Point Sensor`